xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecAndroidMediaCodec.cpp

```diff
@@ -499,7 +499,9 @@ bool CDVDVideoCodecAndroidMediaCodec::Open(CDVDStreamInfo &hints, CDVDCodecOptio
       m_formatname = "amc-hevc";
 
       bool isDvhe = (m_hints.codec_tag == MKTAG('d', 'v', 'h', 'e'));
-      bool isDvh1 = (m_hints.codec_tag == MKTAG('d', 'v', 'h', '1'));
+      bool isDvh1 = (m_hints.codec_tag == MKTAG('d', 'v', 'h', '1') ||
+                     m_hints.codec_tag == MKTAG('h', 'v', 'c', '1') ||
```

Comment:
because of this line this file iPhone11_4K-recorder_59.940HDR10.mov which is number 13 from Kodi samples will try to use dolby-vision codec even if it's not a DV file. It has a hvc1 tag.

Format profile : Main 10@L5.1@High
Codec ID         : hvc1

from kodi logs:
> CDVDVideoCodecAndroidMediaCodec:: Open Android MediaCodec OMX.amlogic.dolby-vision.decoder.awesome

It still plays as HDR on Ugoos AM6 Plus device but I'm not sure about other devices that don't have dolby vision codec.

Reply:
I figure this was needed since in the code you posted this would cause isDvh1 to be true and would cause the same effect. Dolby Vision decoder to be used. 

Now what triggers Dolby Vision is:

- Presence of side_data
- dvhe tag
- dvh1 tag

Reply:
that's why I put hvc1 tag check only after I know that Stream Type is Dolby Vision.

Reply:
The result is the same. As the if statement just below the tags we are discussing both will set the m_mime to video/dolby-vision. 

There's no difference in amc-dvhe and amc-dvh1 for m_formatname to what the decoder will use, both use the Dolby Vision decoder.